run-test.sh

```diff
@@ -214,6 +172,8 @@ runtest()
   then
     rm mscorlib.ni.dll
   fi
+
+  chmod +x ./corerun
```

Comment:
Just out of curiosity, why is this necessary? On which platformsis `corerun` not already executable?